database/migration/schema/V7__create_procedure_insert_audience.sql

```diff
@@ -0,0 +1,23 @@
+DROP PROCEDURE IF EXISTS insert_audience;
+DELIMITER //
+CREATE PROCEDURE insert_audience(IN room_id INT, IN start_time INT)
+
+BEGIN
+  DECLARE i INT DEFAULT start_time DIV 60000;
+  DECLARE end_time INT DEFAULT 0;
+  DECLARE end_times CURSOR FOR
```

Comment:
`end_times` looks like a collection. Please change variable name so it's easier to understand it is a cursor.